test/4_test.py

```diff
@@ -0,0 +1,315 @@
+import logging
+import requests
+import uuid
+import urllib
+import sys
+import time
+
+class Test:
```

Comment:
> I'm not sure if it would be helpful, but was it considered to make this a unittest testcase?

I assume that `unittest` is a library, right?

I could certainly do this, if it's more standard. So far, I've been operating blindly, (re)learning Python, Docker and Buildkite in the dark, but I'll be happy to improve readability :)

Reply:
[unittest is the standard library module](https://docs.python.org/3/library/unittest.html) for testing in Python, I'm not sure if this code fits well into the assumptions it makes though, to be honest. It probably isn't worth messing with this right now.